firebase/firestore.rules

```diff
@@ -4,5 +4,8 @@ service cloud.firestore {
     match /alerts-subscriptions/{email} {
       allow write;
     }
+    match /snapshots/{snapshot_id}/locations/{fips}/emails/{email} {
+      allow write;
+    }
```

Comment:
Per my other comments, I think we want to revert this, redeploy the old rules, and switch to using the admin SDK.